build.gradle

```diff
@@ -20,7 +20,7 @@ apply plugin: 'war'
 
 war {
     baseName = 'remrem-generate'
-    version =  '0.5.9'
+    version =  '0.5.10'
```

Comment:
I think this will be 0.6.2 based on current master branch. Also miss change to CHANGELOG.md